cppcache/test/AttributesFactoryTest.cpp

```diff
@@ -23,36 +23,40 @@ using namespace apache::geode::client;
 
 TEST(AttributesFactoryTest, setEntryIdleTimeoutSeconds) {
   AttributesFactory attributesFactory;
-  attributesFactory.setEntryIdleTimeout(ExpirationAction::DESTROY,
-                                        std::chrono::seconds(10));
   std::shared_ptr<RegionAttributes> regionAttributes =
-      attributesFactory.createRegionAttributes();
+      attributesFactory
```

Comment:
Similar to the comments above I would add a test that makes sure the the return from each set call is a reference to the same object we called the set on.